### PR TITLE
create pages for each relevant law

### PR DESCRIPTION
**Pull request by afeld**
Only made minimal changes to the content itself – just split it up to keep it organized.

### CODE REVIEW

File: _pages/laws/508.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: "508"
+---
+
+_Bristow and Yuda are good contacts for this and they are developing training._ See the [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/) for a checklist.
```
Comment (DavidEBest) [inline _pages/laws/508.md]: Could we pull individual names out the guide, and point people to other guides and Slack channels?

**Comment by DavidEBest** [comment]
@afeld Looks pretty good, but there is a merge conflict that needs to be cleared up.

File: _pages/laws/508.md (hunk 1 above)
Comment (afeld) [inline _pages/laws/508.md]: Totally agree – I was just copying-and-pasting this from the other page.

**Comment by afeld** [comment]
Updated!

**Pull request merged**